spec/integration/css_helper_spec.rb

```diff
@@ -39,6 +39,15 @@ def expect_file(path, content='file content')
         expect(css_for_doc(doc)).to eq("content of base.css\ncontent of font.css")
       end
     end
+
+    context 'when HTML contains ignored links' do
+      let(:files) { ["stylesheets/ignore.css' data-premailer='ignore'"] }
```

Comment:
Yeah, this is super hacky. :sunglasses: I updated it to use a rails-y last argument as options hash, though I'd be happy to change it.